### DISCUSSION TITLE
Need to load modules in a new pic session

### DISCUSSION THREAD

**Discussion by leeyap** [JGCRI/gcam-core#212] [Installing and running GCAM]
I opened a new session in Putty and tried to enter R. I got the following error:
`bash: R: command not found...`

I had forgotten to load in the necessary modules, which fixed the issue.

```
module load git
module load svn/1.8.13
module load R/3.3.3
module load java/1.8.0_31
module load gcc/6.1.0
```

**Comment by marideeweber** [comment]
You can also add these modules to your user specific aliases and functions if you don't want to load them every time. To do this in WinSCP you go to **Options > Preferences > Panels > Show Hidden Files** and then navigate to _root/qfs/people/your username_. Open the _.bashrc_ file, and under #User specific aliases and functions you can paste the list of modules.
